src/main/resources/python/security_schemes.hbs

```diff
@@ -156,6 +212,9 @@ class HTTPDigestSecurityScheme(__SecuritySchemeBase):
         body: typing.Optional[typing.Union[str, bytes]],
         query_params_suffix: typing.Optional[str],
         scope_names: typing.Tuple[str, ...] = (),
+{{#if oauthServerHostnames}}
+        oauth_server_client_info: OauthServerClientInfo = {}
```

Comment:
Mutable defaults (like a dict) should not be used in functions.
Please move this input above scope names and remove the default value.

Reply:
Fixed